src/content/release-notes/waf.yaml

```
@@ -5,11 +5,14 @@ productLink: "/waf/"
 productArea: Application security
 productAreaLink: /fundamentals/reference/changelog/security/
 entries:
-  - publish_date: "2025-06-09"
-    scheduled_date: "2025-06-16"
+  - publish_date: "2025-06-16"
+    scheduled_date: "2025-06-23"
```

```suggestion
  - publish_date: "2025-06-09"
    scheduled_date: "2025-06-16"
```
We usually only update the dates when adding new scheduled releases, not when clearing data.